test/backend-test/test-mqtt.js

```diff
@@ -52,7 +54,7 @@ async function testMqtt(mqttSuccessMessage, mqttCheckType, receivedMessage) {
 }
 
 describe("MqttMonitorType", {
-    concurrency: true,
+    concurrency: 4,
```

Comment:
What is the background behind this change? How was 4 chosen? 🤔

Reply:
I almost doubled the number of tests in this `describe`. For each test, a HiveMQ docker container is started. With  `concurrency: true`, they are all started at the same time.

My powerful laptop and servers didn't like too much starting 13 containers at exactly the same time. The test suite needed over 2 minutes, and sometimes didn't even return at all. With `concurrency: 4`, the suite consistently required ~22s, which cannot get much better with a 20s timeout. There are 3 timeout tests, so `concurrency: 3` would probably be fine as well. Anything more than 4 and I'd be afraid to DDOS other computers when testing.